source/app/controllers/admin_controller.rb

```diff
@@ -1,4 +1,9 @@
 class AdminController < ApplicationController
   def show
+  	user = User.find(session[:current_user_id]) if logged_in?
```

Comment:
The `current_user` method from `ApplicationController` is available here, since this is a subclass of `ApplicationController`.